x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/queries/helpers.ts

```diff
@@ -9,7 +9,7 @@ import type { ESQLAstQueryExpression, ESQLCommand } from '@kbn/esql-ast';
 import { Walker, BasicPrettyPrinter, isFunctionExpression, isColumn, mutate } from '@kbn/esql-ast';
 import type { DataViewFieldMap } from '@kbn/data-views-plugin/common';
 import { partition } from 'lodash/fp';
-import type { ESQLProperNode } from '@kbn/esql-ast/src/types';
+import type { ESQLForkParens, ESQLProperNode } from '@kbn/esql-ast/src/types';
```

Comment:
1) Maybe I should create something backwards compatible utils that accepts branches 🤔 

2) or keep this fix is enough